lib/discourse_code_review/state/commit_topics.rb

```diff
@@ -209,6 +209,12 @@ def find_linked_commits(text)
 
         result
       end
+
+      def escape_trailers(body)
+        return body if body !~ /\n\n(?:.*[:=].*\n?)+.\z/mi
```

Comment:
A few questions:

- Can trailers be separated by `=` ?
- Can trailers have no content before and/or after the `:` ?
- Why the `.` before the `\z` ? Is it to catch any new lines? If so, maybe use `\s*` instead?